src/microbe_stage/MicrobeAI.cs

```diff
@@ -9,385 +10,179 @@
 /// </summary>
 /// <remarks>
 ///   <para>
-///     This is ran in a background thread so no state changing or scene spawning methods on Microbe may be called.
+///     This is run in a background thread so no state changing or scene spawning methods on Microbe may be called.
 ///   </para>
 /// </remarks>
 public class MicrobeAI
 {
-    private readonly Compound atp;
-
-    // ReSharper disable once NotAccessedField.Local
-    private readonly Compound iron;
+    private readonly Compound glucose;
     private readonly Compound oxytoxy;
+    private readonly Compound ammonia;
+    private readonly Compound phosphates;
 
     [JsonProperty]
     private Microbe microbe;
 
-    [JsonProperty]
-    private int boredom;
-
     // ReSharper disable once CollectionNeverQueried.Local
     [JsonIgnore]
     private List<FloatingChunk> chunkList = new List<FloatingChunk>();
 
-    [JsonProperty]
-    private bool hasTargetPosition;
-
-    [JsonProperty]
-    private LifeState lifeState = LifeState.NEUTRAL_STATE;
-
-    [JsonProperty]
-    private bool moveFocused;
-
-    [JsonProperty]
-    private float movementRadius = 2000;
-
-    [JsonProperty]
-    private bool moveThisHunt = true;
-
-    // All of the game entities stored here are probable places where disposed objects come from
-    // so they are ignored for now
-    [JsonIgnore]
-    private Microbe predator;
-
-    // Prey and predator lists
-    [JsonIgnore]
-    private List<Microbe> predatoryMicrobes = new List<Microbe>();
-
     [JsonProperty]
     private float previousAngle;
 
-    [JsonIgnore]
-    private Microbe prey;
-
-    [JsonIgnore]
-    private Microbe previousPrey;
-
-    [JsonIgnore]
-    private List<Microbe> preyMicrobes = new List<Microbe>();
-
-    [JsonIgnore]
-    private bool preyPegged;
-
-    [JsonIgnore]
-    private FloatingChunk targetChunk;
-
-    [JsonIgnore]
-    private FloatingChunk previousChunk;
-
     [JsonProperty]
     private Vector3 targetPosition = new Vector3(0, 0, 0);
 
-    /// <summary>
-    ///   TODO: change to be the elapsed time instead of AI update count
-    /// </summary>
     [JsonProperty]
-    private float ticksSinceLastToggle = 600;
+    private Microbe focusedPrey;
```

Comment:
This must be JsonIgnore, see: https://github.com/Revolutionary-Games/Thrive/issues/2029